### PR TITLE
Feature/sitecore 10

### PR DESCRIPTION
**Pull request by mcreale**
This PR contains updates to: 

- add Sitecore 10 support
- refactor Cognifide namespaces to Crownpeak.Sitecore.DQM
- refactor Cognifide references in images and text to iMedia

I might recommend that we refactor this a little more to be a Feature in the Sitecore Helix architecture. this will allow developers to more easily incorporate it into their solutions and allow it to be deployed via CI or CD.

Additionally, let me know if you want me to generate a Sitecore package for the release.

Thanks,
Matt

### CODE REVIEW

File: serialization/Crownpeak.Sitecore.DQM/ContentEditorProofingRibbon/Crownpeak DQM.yml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      Hint: Header
+      Value: Validierung
+    - ID: "5dd74568-4d4b-44c1-b513-0af5f4cda34f"
+      Hint: __Created by
+      Value: |
+        sitecore\admin
+    - ID: "a1a31a93-c4a0-4396-b629-67c33001a3cf"
+      Hint: Tooltip
+      Value: Validierungsfehler anzeigen.
+- Language: en
```
Comment (girichards) [inline serialization/Crownpeak.Sitecore.DQM/ContentEditorProofingRibbon/Crownpeak DQM.yml:76]: just noting that this `en` section looks like the only one without a `__Display name` block. I don't know Sitecore/plugins well enough to know if this is significant 

```
  Fields:
  - ID: "b5e02ad9-d56f-4c41-a065-a133db87bdeb"
    Hint: __Display name
    Value: XXXXXXXXXXXXXXX
```

**Review by girichards — APPROVED** [review]
looks ok, though I don't really know Sitecore and plugin development well enough pick out any issues

File: src/Cognifide.Sitecore.Crownpeak/App_Config/Include/Crownpeak.Sitecore.config [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+      <command name="crownpeakdqm:previewtab" type="Crownpeak.Sitecore.DQM.Commands.ShowPreviewTab, Crownpeak.Sitecore.DQM" />
     </commands>
     <settings>
-      <setting name="Crownpeak.ApiKey" value="AEJzaS2jn04uwEdbVLTtI4jj1nkyYc3r2OE68Uxl" />
-      <setting name="Crownpeak.WebsiteId" value="809d0b9535b47a7cd700b1df88794955" />
+      <setting name="Crownpeak.ApiKey" value="{{API_KEY}}" />
+      <setting name="Crownpeak.WebsiteId" value="{{WEBSITE_ID}}" />
       <setting name="Crownpeak.RetryCount" value="5" />
 
       <!--Put value = 0 to stop loggng otherwise put value = 1-->
```
Comment (ianhaizlip) [inline src/Cognifide.Sitecore.Crownpeak/App_Config/Include/Crownpeak.Sitecore.config:11]: doesn't matter, but loggng? logging.

**Review by ianhaizlip — APPROVED** [review]

**Comment by danhanley** [comment]
I really should unsubscribe from this project :-)

**Pull request merged**